client/src/Clusters/Footer/data.js

```diff
@@ -0,0 +1,20 @@
+export const data = {
+    title: "Google Play",
+    categories: [
+        {
+            id: "01",
+            title: "Google Play",
+            subcategories: ["Play Pass", "Play Points", "Gift Cards", "Refund Policy"]
```

Comment:
I would format this a bit differently. You should put the mock url here (in this case, `#`).

Reply:
The URL should be inside each subcategory. They just happen to all be "#", but in reality, they can be different URLs.

I.e.:
``` js
subcategories: [
  {
    title: "Play Pass",
    url: "#"
  },
  {
    title: "Play Points",
    url: "#"
  }
]
```

Reply:
Doesn't look resolved